src/Lecture1.hs

```diff
@@ -77,7 +87,12 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+
+-- Is it ok to ignore "Pattern match(es) are non-exhaustive"
+-- if its guarantied that it's exhaustive like in this case?
```

Comment:
In this case, it's okay. But generally, it's better to avoid non-exhaustive pattern matchings. You never know if someone else will refactor the code and introduce an error accidentally.